extra-gnome/caribou/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=caribou
+PKGDES="A text entry and UI navigation application (on-screen keyboard)"
+PKGSEC=gnome
+PKGDEP="at-spi2-atk clutter dconf gtk-3 libgee libxklavier pyatspi pygobject-3"
+BUILDDEP="gtk-2 intltool docbook-xsl"
+
+
+NOLTO=1
```

Comment:
Why?

Reply:
Oh, probably was left over while I experimenting building this package.